ConwaysGameOfLife/Messages.cs

```diff
@@ -10,6 +10,6 @@ public class Messages
 
         public const string QuitGame = "Enter 'q' to quit Game.";
 
-        public const string RequestLiveCells = "Please enter the x, y coordinates for Live cells."; // TODO: format of string?
+        public const string RequestLiveCells = "Please enter the x, y coordinates for one live cell."; // TODO: format of string?

```

Comment:
could be worth giving the user an example? 
```suggestion
        public const string RequestLiveCells = "Please enter the coordinates for one live cell e.g. 0,1."; // TODO: format of string?

```